README.md

```diff
@@ -1,6 +1,8 @@
 # xhr
 
-A small XMLHttpRequest wrapper. Designed for use with [browserify](http://browserify.org/).
+A small XMLHttpRequest wrapper. Designed for use with [browserify](http://browserify.org/), [webpack](https://webpack.github.io/) etc.
+
+API is a subset of [request](https://github.com/request/request) so that the same code can be used in the browser and Node.js.
```

Comment:
I'd mention that the streaming API has been removed; size reasons and stuff :sparkles: